src/app/plan/components/Wizard/WizardComponent.tsx

```diff
@@ -173,7 +187,33 @@ const WizardComponent = (props: IOtherProps & FormikProps<IFormValues>) => {
             </WizardStepContainer>
           ),
           canJumpTo: stepIdReached >= stepId.StorageClass,
-          nextButtonText: 'Finish',
+        },
+        {
+          id: stepId.Hooks,
+          name: 'Hooks',
+          component: (
+            <HooksStep
```

Comment:
If you wrap this in `<WizardStepContainer title="Hooks">` you don't need the title text inside HooksStep.